tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -52,3 +58,38 @@ def dismiss_tooltip(self):
         confirm_element = Wait(self.marionette).until(expected.element_present(*self._button_popup_intro))
         Wait(self.marionette).until(expected.element_displayed(confirm_element))
         confirm_element.tap()
+
+class BugzillaLiteStage(BugzillaLite):
+
+    name =  'Bugzilla Lite Stage'
+    manifest_url = 'http://bzlite-staging.herokuapp.com/manifest.webapp'
+
+    def launch(self):
+        Base.launch(self)
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._popup_intro))))
+
+    def dismiss_content(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._content_locator))))
+        close_element = self.marionette.find_element(*self._close_locator)
+        close_element.tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._login_form_locator))
+
+    def create_new_bug(self, title, description):
+        self.marionette.find_element(*self._create_button).tap()
+        from gaiatest.apps.bzlite.regions.create_bug import CreateBug
+        new_bug = CreateBug(self.marionette)
+        new_bug.tap_new_bug(title, description)
+        return new_bug
+
+    @property
+    def is_loaded(self):
+        self.wait_for_bugpage_to_be_displayed()
+        return True
+
+    def wait_for_bugpage_to_be_displayed(self):
```

Comment:
Perhaps it's better to call this "wait_for_page_to_load", because I think that's what it is, right?